validation/provisioning/rke2/custom_test.go

```diff
@@ -0,0 +1,139 @@
+//go:build recurring
```

Comment:
For the build tag, let's make sure we have the choice of `validation` as well. So should be `//go:build validation || recurring`.